projects/ngx-datatable/src/lib/components/body/body.component.ts

```diff
@@ -113,7 +113,15 @@ import { translateXY } from '../../utils/translate';
         >
         </datatable-summary-row>
       </datatable-scroller>
-      <div class="empty-row" *ngIf="!rows?.length && !loadingIndicator" [innerHTML]="emptyMessage"></div>
+      <div
+        class="empty-row"
+        *ngIf="!rows?.length && !loadingIndicator && !emptyCustomContent"
+        [innerHTML]="emptyMessage"
+      ></div>
+      <ng-content
+        select="[empty-content]"
+        *ngIf="!rows?.length && !loadingIndicator && emptyCustomContent"
+      ></ng-content>
```

Comment:
shame really that Angular _still_ doesn't do default content like Vue: https://github.com/angular/angular/issues/12530 🙄